ValorantClient.py

```diff
@@ -0,0 +1,137 @@
+import datetime
+import re
+import time
+from aiohttp import ClientSession
+import requests
+
+
+class ValorantClient(ClientSession):
+    """
+    Emulates Valorant PC Client 
+    """
+    def __init__(self):
+        super().__init__()
+        # Set of headers that Riot Expects from a Val Client to pass with each request
+        # Can be captured using Fiddler Capture
+        self.riot_headers = {
+            "User-Agent": "RiotClient/43.0.1.4195386.4190634 rso-auth (Windows; 10;;Professional, x64)",
+            "X-Riot-ClientVersion": ValorantClient.get_client_version(),
+            # X-Riot-ClientPlatform = base64 string to emulate the following:
+            # {
+            #     "platformType": "PC",
+            #     "platformOS": "Windows",
+            #     "platformOSVersion": "10.0.19042.1.256.64bit",
+            #     "platformChipset": "Unknown"
+            # } 
+            "X-Riot-ClientPlatform": "ew0KCSJwbGF0Zm9ybVR5cGUiOiAiUEMiLA0KCSJwbGF0Zm9ybU9TIjogIldpbmRvd3MiLA0KCSJwbGF0Zm9ybU9TVmVyc2lvbiI6ICIxMC4wLjE5MDQyLjEuMjU2LjY0Yml0IiwNCgkicGxhdGZvcm1DaGlwc2V0IjogIlVua25vd24iDQp"
+        }
+
+    # Gets up-to-date version from 3rd party API
+    @staticmethod
+    def get_client_version():
```

Comment:
if this is a static method, why not just move this out of this Valorant Client Session class, and into a helper function